packages/theme/src/borders.ts

```diff
@@ -20,6 +20,7 @@ export const nativeBorders = {
         lg: 2,
     },
     radii: {
+        checkbox: 4,
```

Comment:
I think it will be better to move this to checkbox component directly. We don't need to have access to this in global theme.

Or maybe we can rename it to something like "small" to make it universal, but I am not sure if we will even need it again.